url-updater/src/main/java/com/devonfw/tools/ide/url/updater/gradle/GradleUrlUpdater.java

```diff
@@ -30,8 +43,52 @@ protected String getTool() {
   @Override
   protected void addVersion(UrlVersion urlVersion) {
 
-    doAddVersion(urlVersion, "https://services.gradle.org/distributions/gradle-${version}-bin.zip");
+    String responseBody = doGetResponseBodyAsString(HASH_VERSION_URL);
```

Comment:
functionally correct. But we are requesting this HTML for every version what seems to be waste.
Can we do that only once? The easiest way would be to use a lazy getter with a member variable for the HTML (currently `responseBody`).

Reply:
I've adjusted it now.